calyx-py/test/correctness/queues/strict_and_rr_queues/gen_strict_or_rr.py

```diff
@@ -140,7 +150,7 @@ def insert_strict_pifo(
                 ],
             ),
             len_decr,
-            restore_hot,  # Peeking must not affect `hot`, so we restore it.
+            restore_hot if not round_robin else ast.Empty            
```

Comment:
Did the `[]` trick not work? I remain kinda interested in not importing `ast`.

Reply:
The [] ended up not working, but ast did. When we ran it together, it was only on one strict test.

Reply:
Gotcha, thanks for trying!